Problem-3

```diff
@@ -0,0 +1,45 @@
+// Time Complexity :O(log(n))
+// Space Complexity :O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : No
+
+
+// Your code here along with comments explaining your approach
```

Comment:
Problem 3 is Finding a peak element. You submitted the problem which is part of Binary-search 1 Repo.

Reply:
Please re-submit Find Peak element again